widget/view.controller.js

```diff
@@ -228,12 +232,25 @@
                     }
                 ]
             };
-            $resource(API.QUERY + 'solutionpacks').save({ $limit: ALL_RECORDS_SIZE }, queryBody).$promise.then(function (response) {
-                if (response['hydra:member'] || response['hydra:member'].length > 0) {
-                    var huntToolDetails = _.map(response['hydra:member'], obj => _.pick(obj, ['name', 'label', 'version', 'uuid']));
-                    _loadConnectorDetails(huntToolDetails[0].name, huntToolDetails[0].version, huntToolDetails[0]);
-                }
-            });
+            $resource(API.QUERY + 'solutionpacks').save({ $limit: ALL_RECORDS_SIZE }, queryBody).$promise
```

Comment:
can this be part of service code?

Reply:
we are not used service here so added here

Reply:
introduce one later, if same case arises